assign-user/action.yml

```diff
@@ -0,0 +1,46 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+name: 'assign-user'
+description: 'Assign user to a target Power Platform environment with specified security role'
+inputs:
+
+  environment:
+    description: 'Id or URL of Power Platform environment to assign user to.'
+    required: true
+
+  user:
+    description: 'Azure AD Object ID or user''s principal name to assign to the target environment'
+    required: true
+
+  role:
+    description: 'Id of the security role to assign user'
+    required: true
+
+  user-name:
+      description: 'Power Platform user name to authenticate with, e.g. myname@my-org.onmicrosoft.com. Setting this input makes user-name and password required; specifying alternate "app-id" credential set of inputs will result in an error.'
+      required: false
+
+  password-secret:
+    description: 'Power Platform password, required if authenticating with username. Do NOT checkin password, instead create a secret and reference it here with: see: https://help.github.com/en/actions/configuring-and-managing-workflows/creating-and-storing-encrypted-secrets#using-encrypted-secrets-in-a-workflow'
+    required: false
+
+  app-id:
+    description: 'The application id to authenticate with. Setting this input makes app-id, tenant-id and client-secret required; specifying alternate "username" credential set of inputs will result in an error.'
+    required: false
+
+  client-secret:
+    description: 'The client secret to authenticate with. Required if authenticating with app-id.'
+    required: false
+
+  tenant-id:
+    description: 'Tenant id if using app-id & client secret to authenticate.'
+    required: false
+
+  cloud:
```

Comment:
remove for now, should be addressed with AB#2871196

Reply:
tagging to the separate work item: AB#2871196

Reply:
Darn thought I did. Must have pruned it from the other PR and not this one.